app/components/progress_bar_component/progress_bar_component.html.erb

```diff
@@ -1,32 +1,62 @@
-    </h4>
-    <p class="govuk-body-s ncce-progress-section--description">
-    <strong>
-      <% if @body.present? %>
-        <%= @body %>
-      <% else %>
-        Complete the below training and activities to receive your <%= @programme.title %> certificate.
-        You can complete these at your own pace and in any order.
-      <% end %>
-    </strong>
-    </p>
-    <% if @steps_to_accreditation %>
-      <p class="govuk-body-s">Choose your first step to certification:</p>
-    <% end %>
-    <div class="ncce-progress-section">
-      <% @programme_objectives.each_with_index do |programme_objective, index| %>
-        <% class_name = programme_objective&.user_complete?(@current_user) ? "icon-ticked-circle" : "icon-blank-circle" %>
-        <a href="<%= programme_objective.progress_bar_path %>" class="<%= class_name %> ncce-link ncce-progress-section--text">
-          <span><%= programme_objective.progress_bar_title.html_safe %></span>
-        </a>
+<%= render GovGridRowComponent.new(padding: {bottom: 0}) do |row|  %>
+  <%= row.with_column(@programme.show_extra_objectives_on_progress_bar? ? "two-thirds" : "full") do %>
+    <div class="progress-bar-component">
+      <h2 class="govuk-heading-m"><%= @title.presence || 'Progress bar' %></h2>
+      <p class="govuk-body">
+        <% if @body.present? %>
+          <%= @body %>
+        <% elsif @programme.show_extra_objectives_on_progress_bar? %>
+          The key steps to earn your <%= @programme.title %> certificate</p>
+        <% else %>
+          Complete the below training and activities to receive your <%= @programme.title %> certificate.
+          You can complete these at your own pace and in any order.
+        <% end %>
+      </p>
+      <div class="<%= content_spacing_class("progress-bar-component__objectives-wrapper") %>">
+        <% if @programme.show_extra_objectives_on_progress_bar? %>
+          <div class="progress-bar-component__objective">
+            <div class="progress-bar-component__objective--icon <%= user_enrolled_class %>">
+            </div>
+            <div class="<%= content_spacing_class("progress-bar-component__objective--text") %>">
+              <p class="govuk-body-s govuk-!-margin-0">Enrol</p>
+            </div>
+          </div>
+          <div class="progress-bar-component--dotted-line icon-dotted-line"></div>
+        <% end %>
 
-        <% unless index == @programme_objectives.size - 1 %>
-          <div class="icon-dotted-line"></div>
+        <% @programme_objectives.each_with_index do |programme_objective, index| %>
+          <% if programme_objective.multi_stage_group? %>
+            <% multi_stage_objectives.each_with_index do |objective, index| %>
+              <div class="progress-bar-component__objective">
+                <div class="progress-bar-component__objective--icon <%= course_bookings_status_class(programme_objective, objective[:state]) %>"></div>
```

Comment:
Could you include the hardcoded class in the response from course_bookings_status_class method, just makes this a little bit more readable?